xhttp/client.go

```diff
@@ -158,7 +159,26 @@ func (r *retrierClient) do(ctx context.Context, req *http.Request, requestBody [
 		}
 		log.Debug("xhttp.Client: retrying request with error status code")
 		r.onRetry(req, res, err)
-		// Maybe add handling for Retry-After header, so far this seems to be enough
+
+		// handle Retry-After header
+		const minSleepPeriod = time.Second
```

Comment:
```suggestion
		const minRetryAfterDuration = time.Second
```

It could be mistaken with some sort of min/max sleep duration we have on the retrier (we have a max one..but no min...but it got me thinking it could be interpreted as that).